x-pack/plugins/maps/public/classes/layers/icons/top_hits_layer_icon.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+
+export const TopHitsLayerIcon: FunctionComponent = () => (
```

Comment:
I like it, but maybe was hoping we could make it stand out from the tracks-icon more. Can be separate PR.

cc @miukimiu